modin/pandas/series.py

```diff
@@ -678,9 +678,7 @@ def f(x):
             else:
                 return result
         else:
-            result = getattr(sys.modules[self.__module__], return_type)(
-                query_compiler=result
-            )
+            result = globals()[return_type](query_compiler=result)
```

Comment:
Is this change just because the code is shorter now, or is there some other reason?

Reply:
I have changed the logic of accessing the globals of the module to be more direct (and it is also slightly faster, though I doubt we would be able to measure the difference)